package.json

```diff
@@ -12,6 +12,7 @@
     "prom-client": "^11.1.3",
     "underscore": "~1.6.0",
     "@google-cloud/debug-agent": "^3.0.0",
+    "@google-cloud/profiler": "0.2.3",
```

Comment:
Is there a reason for this being 0.2.3 rather than ^0.2.3?

Reply:
added ^